ui/src/views/settings/SettingsRelay.vue

```diff
@@ -314,11 +411,98 @@ export default {
       );
     },
     setRelayConfigurationCompleted() {
+      // set always bcc
+      this.setAlwaysBcc();
+    },
+    async setAlwaysBcc() {
+      this.loading.setRelayConfiguration = true;
+      this.error.setRelayConfiguration = "";
+      this.error.bcc = "";
+      const taskAction = "set-always-bcc";
+      const eventId = this.getUuid();
+
+      // register to task error
+      this.core.$root.$once(
+        `${taskAction}-aborted-${eventId}`,
+        this.setAlwaysBccConfigurationAborted
+      );
+
+      // register to task validation
+      this.core.$root.$once(
+        `${taskAction}-validation-failed-${eventId}`,
+        this.setAlwaysBccConfigurationValidationFailed
+      );
+
+      // register to task completion
+      this.core.$root.$once(
+        `${taskAction}-completed-${eventId}`,
+        this.setAlwaysBccConfigurationCompleted
+      );
+
+      const res = await to(
+        this.createModuleTaskForApp(this.instanceName, {
+          action: taskAction,
+          data: {
+            bcc: this.isBccEnabled ? this.bcc : "",
+          },
+          extra: {
+            title: this.$t("action." + taskAction),
+            eventId,
```

Comment:
In this specific case, let's hide progress notification with `isNotificationHidden: true`, or the users will see two notifications when they click Save button; that would be a bit strange